docker/Dockerfile

```diff
@@ -45,6 +45,7 @@ ARG CUDA_VER
 # Install dependencies to build
 RUN apt-get update &&\
     apt-get upgrade -y &&\
+    DEBIAN_FRONTEND=noninteractive apt-get install -y openjdk-8-jdk openjdk-8-jre &&\
```

Comment:
Do we need both the JRE and JDK, or will just the JRE work? Also, let's update this to use the version `11` to match with what we're using for development, and use `headeless` to keep the footprint small.
```suggestion
    DEBIAN_FRONTEND=noninteractive apt-get install -y openjdk-11-jre-headless &&\
```

Reply:
sure, will make those changes, i'm testing it with only jre atm, will update when i've things working

Reply:
@cwharris made changes in new commit, please review